docs/hub/config.md

```diff
@@ -47,7 +50,7 @@ bCd7nPuNAyYHCOOHAgMBAAE=
 
 Bash
 ```
-JWT_KEY=`cat jwt_key.pub` ./mecure
```

Comment:
This line of code shows how to use file instead of env variable.

Reply:
Thanks, I'm aware, but that doesn't work with docker-compose, or in my second attempt with Intellij (the problem here was that Intellij replaces newline with space, and golang's pem.Decode failes if the key doesn't have newlines in it in the regular format)

Reply:
did you try to change the command (within docker-compose) in `CMD: 'JWT_KEY=$$(cat jwt_key.pub) mercure'` (maybe some things need to be escaped) or to add `export JWT_KEY=$(cat jwt_key.pub)` in the entrypoint,

Reply:
Good idea and I just tried it, but since the finale mercure image is built from the `scratch` image, it doesn't have the cat binary :(

Reply:
Also, I feel Mercure should be able to load a key from a file, and resorting to bash "hacks" should be avoided :)